CHANGES.md

```diff
@@ -51,6 +52,85 @@
 * [Version 1.0.1](#version-101)
 * [Version 1.0](#version-10)
 
+## Version 3.1.2 (September 30, 2021)
+
+Patch release with various bug fixes, build improvements, and
+documentation updates.
```

Comment:
I feel like the documentation is worth highlighting a little in order to encourage people to look at it, I think this the first iteration with your large doxygen makeover? Also the deep pixels document in rst format is an accessibility improvement.